photon-client/src/components/dashboard/tabs/ArucoTab.vue

```diff
@@ -20,37 +38,84 @@ const interactiveCols = computed(
     />
     <cv-slider
-      v-model="currentPipelineSettings.numIterations"
+      v-model="currentPipelineSettings.threshConstant"
       class="pt-2"
       :slider-cols="interactiveCols"
-      label="Corner Iterations"
-      tooltip="How many iterations are going to be used in order to refine corners. Higher values are lead to more accuracy at the cost of performance"
-      :min="30"
-      :max="1000"
-      :step="5"
-      @input="(value) => useCameraSettingsStore().changeCurrentPipelineSetting({ numIterations: value }, false)"
+      label="Thresh Constant"
+      tooltip="Affects the threshold window mean value cutoff for all steps. Higher values can improve performance, but may harm detection rate."
+      :min="0"
+      :max="128"
+      :step="1"
+      @input="(value) => useCameraSettingsStore().changeCurrentPipelineSetting({ threshConstant: value }, false)"
+    />
+    <cv-switch
+      v-model="currentPipelineSettings.debugThreshold"
+      class="pt-2"
+      label="Debug Threshold"
+      tooltip="Display the first threshold step to the color stream."
+      @input="(value) => useCameraSettingsStore().changeCurrentPipelineSetting({ debugThreshold: value }, false)"
+    />
+    <!-- <cv-switch
+      v-model="currentPipelineSettings.useAruco3"
+      class="pt-2"
+      label="Use ArUco3 Speedup"
+      tooltip="Enables an 'ArUco3' implementation which may increase performance at the cost of detection distance. This is similar to AprilTag's 'decimation'."
+      @input="(value) => useCameraSettingsStore().changeCurrentPipelineSetting({ useAruco3: value }, false)"
     />
     <cv-slider
-      v-model="currentPipelineSettings.cornerAccuracy"
+      v-model="currentPipelineSettings.aruco3MinMarkerSideRatio"
       class="pt-2"
       :slider-cols="interactiveCols"
-      label="Corner Accuracy"
-      tooltip="Minimum accuracy for the corners, lower is better but more performance intensive "
-      :min="0.01"
-      :max="100"
-      :step="0.01"
-      @input="(value) => useCameraSettingsStore().changeCurrentPipelineSetting({ cornerAccuracy: value }, false)"
+      label="ArUco3 Min Marker Ratio"
+      tooltip="Minimum side length of markers expressed as a ratio of the largest image dimension."
+      :min="0"
+      :max="0.1"
+      :step="0.001"
+      @input="
+        (value) => useCameraSettingsStore().changeCurrentPipelineSetting({ aruco3MinMarkerSideRatio: value }, false)
+      "
     />
+    <cv-slider
+      v-model="currentPipelineSettings.aruco3MinCanonicalImgSide"
+      class="pt-2"
+      :slider-cols="interactiveCols"
+      label="ArUco3 Min Canonical Side"
+      tooltip="Minimum side length of the canonical image (marker after undoing perspective distortion)."
+      :min="16"
+      :max="128"
+      :step="1"
+      @input="
+        (value) => useCameraSettingsStore().changeCurrentPipelineSetting({ aruco3MinCanonicalImgSide: value }, false)
+      "
+    /> -->
```

Comment:
Is there a reason this is all commented out? If it isn't used, it should be removed

Reply:
> I removed the Aruco3 decimation in the UI since it didn't behave how I expected and was pretty confusing.

Just for ease of testing if need be in the future, removing it is fine.